### PR TITLE
Added calculate score in vulnerability metadata

### PR DESCRIPTION
**Pull request by Cheshiriks**

### CODE REVIEW

File: save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt [hunk 1]
```diff
@@ -84,6 +90,18 @@ class VulnerabilityMetadataService(
             latestCosvFile = cosvFile,
         )
 
+        private fun getScore(severity: Severity): Float {
+            val score = calculateBaseScore(severity.score)
+            severity.scoreNum?.let { scoreNum ->
+                if (score != scoreNum.toFloat()) {
+                    logger.warn {
+                        "Score num $scoreNum from osv file not equal to the calculated value from the base score vector $score."
```
Comment (akuleshov7) [inline save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt:98]: IS not equal

**Review by akuleshov7 — APPROVED** [review]

File: save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt]: Fixed

**Review by nulls — APPROVED** [review]

File: save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt [hunk 2]
```diff
@@ -65,6 +69,8 @@ class VulnerabilityMetadataService(
     ): VulnerabilityMetadata? = vulnerabilityMetadataRepository.findByIdentifier(identifier)
 
     companion object {
+        @JvmStatic
+        private val logger = LoggerFactory.getLogger(VulnerabilityMetadataService::class.java)
```
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt:73]: Changed it

**Review by nulls — APPROVED** [review]

File: save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt [hunk 3]
```diff
@@ -65,6 +69,8 @@ class VulnerabilityMetadataService(
     ): VulnerabilityMetadata? = vulnerabilityMetadataRepository.findByIdentifier(identifier)
 
     companion object {
+        @JvmStatic
+        private val logger = getLogger(VulnerabilityMetadataService::class.java)
```
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt]: Fixed

**Pull request merged**